packages/block-editor/src/store/selectors.js

```diff
@@ -458,22 +458,11 @@ export function getSelectedBlock( state ) {
  *
  * @return {?string} Root client ID, if exists
  */
-export const getBlockRootClientId = createSelector(
-	( state, clientId ) => {
-		const { order } = state.blocks;
-
-		for ( const rootClientId in order ) {
-			if ( includes( order[ rootClientId ], clientId ) ) {
-				return rootClientId;
-			}
-		}
-
-		return null;
-	},
-	( state ) => [
-		state.blocks.order,
-	]
-);
+export const getBlockRootClientId = ( state, clientId ) => {
```

Comment:
I don't know if you have feelings one way or the other, but the stronger convention in this file is to use a function declaration if it's not a memoized selector (mentioning partly because it's easy to overlook when unwrapping as we're doing here).

Reply:
I don't mind changing. My brain doesn't use a consistent method yet :P